### PR TITLE
Add GitHub templates, files for e.g. CoC, and bots e.g. stale-bot

### PR DESCRIPTION
**Pull request by SailReal**
I'm not fully sure about `SUPPORT.md`, if we want to provide other channels for Hub support but we can otherwise still change that in the future.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: software-versions
+    attributes:
+      label: System Setup
+      description: |
+        What software is involved? Please provide version numbers as well.
+      value: |
+          - Hub: [Version shown in the settings of Hub"]
```
Comment (overheadhunter) [inline .github/ISSUE_TEMPLATE/bug.yml:28]:
```suggestion
          - Hub: [Version shown in the settings of Hub]
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          - Keycloak: [Version shown in the settings of Hub]
+          - …
```
Comment (overheadhunter) [inline .github/ISSUE_TEMPLATE/bug.yml:30]:
```suggestion
          - Cryptomator (Desktop/Android/iOS) [Version shown in respective settings]
          - …
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Help & Support
+    url: https://community.cryptomator.org/
+    about: You will find answers in our community forum
```
Comment (overheadhunter) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Should we link to a new Hub section in the forum?

**Review by overheadhunter — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 2 above)
Comment (tobihagemann) [reply .github/ISSUE_TEMPLATE/bug.yml:30]: You forgot a `:` 😁

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (tobihagemann) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Or maybe delete this since it's a "closed" beta?

Comment (overheadhunter) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Well for now the GH interaction limit is in place, so you won't see the issue template chooser at all, I guess.

**Review by tobihagemann — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (SailReal) [reply .github/ISSUE_TEMPLATE/config.yml:5]: We could indeed change the URL to https://community.cryptomator.org/tag/hub but I personally prefer the category view and would wait and see what and how we finally want to offer support for Hub. In our other repos this is also not set and our community seems to get along with it...

**Review by overheadhunter — APPROVED** [review]

**Pull request merged**